gradle.properties

```diff
@@ -77,7 +77,7 @@ graphqlVersion=0.3.0
 protocToolVersion=0.1.0-20230125-180900-96d7ca0
 
 # OpenAPI Module
-openapiVersion=1.4.0-20230201-133700-58d39ab
+openapiVersion=1.4.0-20221220-122000-d6440e1
```

Comment:
Is this correct? Are we releasing openapi with the standard libraries?

Reply:
Fixed with [4e5ce63](https://github.com/ballerina-platform/ballerina-distribution/pull/4046/commits/4e5ce63297194248a125f95f853578952435b529)